aten/src/ATen/core/jit_type.h

```diff
@@ -1550,6 +1551,28 @@ struct CAFFE2_API DeviceObjType : public Type {
   DeviceObjType() : Type(TypeKind::DeviceObjType) {}
 };
 
+struct StreamObjType;
+using StreamObjTypePtr = std::shared_ptr<StreamObjType>;
+// This type represents a Generator
```

Comment:
This is the same comment as from GeneratorType - copy pasta ?

Reply:
Yes. This comment is incorrectly.